src/main/java/io/vertx/core/parsetools/impl/JsonParserImpl.java

```diff
@@ -393,50 +388,42 @@ protected void _closeInput() {
     }
   }
 
-  private class BufferingHandler implements Handler<JsonToken> {
+  private static final EnumMap<JsonEventType, JsonToken> mm = new EnumMap<JsonEventType, JsonToken>(JsonEventType.class);
```

Comment:
Not sure where this is used. Did I miss it?

Reply:
you didn't miss anything, at some point I thought I would need it but finally I did not, so I will clean this up